server/src/main/java/com/objectcomputing/checkins/services/feedback_request_questions/FeedbackRequestQuestion.java

```diff
@@ -0,0 +1,149 @@
+package com.objectcomputing.checkins.services.feedback_request_questions;
+
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.jdbc.annotation.ColumnTransformer;
+import io.micronaut.data.model.DataType;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.annotation.Nullable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "feedback_request_questions")
+public class FeedbackRequestQuestion {
+    @Id
+    @Column(name = "id")
+    @AutoPopulated
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "unique id of the request question answer entry", required = true)
+    private UUID id;
+
+    @Column(name = "requestId")
+    @NotNull
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "id of the feedback request the question answer pair is attached to", required = true)
+    private UUID requestId;
+
+    @Column(name = "questionContent")
+    @NotBlank
+    @TypeDef(type = DataType.STRING)
+    @ColumnTransformer(
+            read =  "pgp_sym_decrypt(questionContent::bytea,'${aes.key}')",
+            write = "pgp_sym_encrypt(?,'${aes.key}')"
+    )
+    @Schema(description = "The question asked to the recipient", required = true)
+    private String questionContent;
+
+    @Column(name = "answerContent")
+    @Nullable
+    @TypeDef(type = DataType.STRING)
+    @ColumnTransformer(
+            read =  "pgp_sym_decrypt(answerContent::bytea,'${aes.key}')",
+            write = "pgp_sym_encrypt(?,'${aes.key}')"
+    )
+    @Schema(description = "The answer to the question", required = false)
+    private String answerContent;
```

Comment:
We may want to make answers its own entity, since in the future it may have a different structure (e.g. multiple choice, checkboxes)